app/clients/zendesk_sell_client.py

```diff
@@ -0,0 +1,94 @@
+import json
+import requests
+
+from typing import Dict, List, Union
+
+from flask import current_app
+
+from app.authentication.bearer_auth import BearerAuth
+from app.user.contact_request import ContactRequest
+
+__all__ = [
+    'ZenDeskSellClient'
+]
+
+
+class ZenDeskSellClient(object):
+
+    def _generate_lead_data(self) -> Dict[str, Union[str, List[str], Dict]]:
+
+        # The API and field definitions are defined here: https://developers.getbase.com/docs/rest/reference/leads
+
+        # validation based upon api mandatory fields
+        assert len(self.contact.name) or len(self.contact.department_org_name), 'Name or Org are mandatory field'
+
+        recipients = {
+            'internal': 'Colleagues within your department (internal)',
+            'external': 'Partners from other organizations (external)',
+            'public': 'Public'
+        }
+        # FIXME: consider migrating to pypi/nameparser for proper name parsing to handle cases like:
+        # 'Dr. Juan Q. Xavier de la Vega III (Doc Vega)'
+        name_tokenised = self.contact.name.split()
+        return {
+            'data': {
+                'last_name': name_tokenised[-1],
+                'first_name': " ".join(name_tokenised[:-1]) if len(name_tokenised) > 1 else '',
+                'organization_name': self.contact.department_org_name,
+                'email': self.contact.email_address,
+                'description': f'Program: {self.contact.program_service_name}\n{self.contact.main_use_case}: '
+                               f'{self.contact.main_use_case_details}',
+                'tags': [self.contact.support_type, self.contact.language],
+                'status': 'New',
+                'custom_fields': {
+                    'Product': ['Notify'],
+                    'Source': 'Demo request form',
+                    'Intended recipients': recipients[self.contact.intended_recipients]
+                    if self.contact.intended_recipients in recipients else 'No value'
+                }
+            }
+        }
+
+    def __init__(self, contact: ContactRequest):
+        self.api_url = current_app.config['ZEN_DESK_SELL_API_URL']
+        self.token = current_app.config['ZEN_DESK_SELL_API_KEY']
+        self.contact = contact
```

Comment:
The `ContactRequest` may be better as an arg of `send_contact_request`. I think that in the near future we'll use this client to send other types of data (when creating a trial service/turning a service live etc). Making the contact form required in the constructor may be a burden